test/prefer-math-trunc.js

```diff
@@ -59,6 +65,32 @@ ruleTester.run('prefer-math-trunc', rule, {
 			errors: errorsBitwise,
 			output: 'const foo = Math.trunc((0, 1.4));'
 		},
+		// Different "types" of 0
+		{
+			code: 'const foo = 1.4 | .0;',
+			errors: errorsBitwise,
+			output: 'const foo = Math.trunc(1.4);'
+		},
+		{
+			code: 'const foo = 1.4 | 0.0000_0000_0000;',
+			errors: errorsBitwise,
+			output: 'const foo = Math.trunc(1.4);'
+		},
+		{
+			code: 'const foo = 1.4 | 0b0;',
+			errors: errorsBitwise,
+			output: 'const foo = Math.trunc(1.4);'
+		},
+		{
+			code: 'const foo = 1.4 | 0x0000_0000_0000;',
```

Comment:
These cases need go to `visualizeTester`, because the message will be `` `| 0x0000_0000_0000 ` ``, different from `` `| 0 ` ``, and it's not tested.

Reply:
You know what, let's move all invalid tests to `visualizeTester`.

Reply:
I'll commit.

Reply:
Ok I'll let you do it